docs/tutorials/hello-tye/00_run_locally.md

```diff
@@ -129,35 +129,42 @@ Now that we have two applications running, let's make them communicate. By defau
     dotnet add frontend/frontend.csproj package Microsoft.Tye.Extensions.Configuration  --version "0.1.0-*"
     ```
 
-5. Now register this client in `Startup.cs` class in `ConfigureServices` of the `frontend` project:
+5. Now register this client in `frontend` by adding the following to the existing `ConfigureServices` method to the existing `Startup.cs` file:
 
    ```C#
+   ...
    public void ConfigureServices(IServiceCollection services)
    {
        services.AddRazorPages();
-
+        /** Add the following to wire the client to the backend **/
        services.AddHttpClient<WeatherClient>(client =>
        {
             client.BaseAddress = Configuration.GetServiceUri("backend");
        });
+       /** End added code **/
    }
+   ...
```

Comment:
What're your thoughts on using diff syntax?

```diff
  ...
  public void ConfigureServices(IServiceCollection services)
  {
      services.AddRazorPages();

+      services.AddHttpClient<WeatherClient>(client =>
+      {
+          client.baseAddress = Configuration.GetServiceUri("backend");
+      });
  }
...
```

Reply:
What I don't love about diff syntax is that it's not easy copy-pastable because of the `+` at the beginning of each line. The fencing with comments, I think makes it pretty clear.